tests/integration/files/conf/master

```diff
@@ -1,3 +1,4 @@
+user: ubuntu
```

Comment:
This is my username and I couldn't run tests as `salt-cloud` always wanted to use `root` and I don't have passwordless sudo. Maybe I should remove this, maybe this needs discussion, I dunno.